packages/tabbed-accordion/index.js

```diff
@@ -1,22 +1,24 @@
 import { AccordionItems } from '@hashicorp/react-accordion'
 import Tabs, { Tab } from '@hashicorp/react-tabs'
+import s from './style.module.css'
 
 function TabbedAccordion({ heading, tabs }) {
   return (
-    <section className="g-tabbed-accordion">
+    <section className={s.root}>
       {heading && (
-        <div
-          data-testid="heading"
-          className="heading-container g-grid-container"
-        >
-          <h2 className="g-type-display-2">{heading}</h2>
+        <div data-testid="heading" className={s.headingContainer}>
+          <h2 className={s.heading}>{heading}</h2>
         </div>
       )}
-      <Tabs className="g-tabs">
+      <Tabs className={heading ? s.tabsAfterHeading : undefined}>
         {tabs.map((tab, stableIdx) => (
           // eslint-disable-next-line react/no-array-index-key
           <Tab key={stableIdx} heading={tab.heading}>
-            <AccordionItems items={tab.items} />
+            <AccordionItems
+              items={tab.items}
+              className={s.accordionItems}
+              withTopBorder={false}
```

Comment:
Added this `withTopBorder` prop to `AccordionItems` as a way to avoid "reach-in" / dangerously brittle CSS overrides.